src/main/java/org/stellar/sdk/BumpFootprintExpirationOperation.java

```diff
@@ -0,0 +1,71 @@
+package org.stellar.sdk;
+
+import lombok.EqualsAndHashCode;
+import lombok.NonNull;
+import lombok.Value;
+import lombok.experimental.SuperBuilder;
+import org.stellar.sdk.xdr.BumpFootprintExpirationOp;
+import org.stellar.sdk.xdr.ExtensionPoint;
+import org.stellar.sdk.xdr.OperationType;
+import org.stellar.sdk.xdr.Uint32;
+
+/**
+ * Represents <a
+ * href="https://developers.stellar.org/docs/fundamentals-and-concepts/list-of-operations#bump-footprint-expiration"
+ * target="_blank">BumpFootprintExpiration</a> operation.
+ *
+ * <p>Bump the expiration of a footprint (read and written ledger keys).
+ *
+ * @see <a href="https://developers.stellar.org/docs/fundamentals-and-concepts/list-of-operations"
+ *     target="_blank">List of Operations</a>
+ */
+@EqualsAndHashCode(callSuper = true)
+@SuperBuilder(toBuilder = true)
+@Value
+public class BumpFootprintExpirationOperation extends Operation {
```

Comment:
can you add TODO's on these that state invoke, bump, restore soroban op classes in this package still require more infrastructure before they can be used in TransactionBuilder and submission, they can't be added as an Operation in a standard 'classic' tx. 

Since they have to go through preflight with rpc first, to define footprint, fees and then build out a single operation tx with all that. the js sdk, js-soroban-client, [has added this functionality for tx sub of soroban ops](https://github.com/stellar/js-soroban-client/pull/108) which can be used as a reference later on for same effort here. I think we should add a new ticket to capture this effort such as 'Submit Soroban Tx operations' which will depend on #485 

I [updated and commented on #486](https://github.com/stellar/java-stellar-sdk/issues/486#issuecomment-1652819596) to reflect the narrowing of scope for this PR to be primarily for the response object marshaling.

Reply:
I have decided not to modify it for now, see https://github.com/stellar/java-stellar-sdk/pull/491#discussion_r1275692913

Reply:
ok, I think understand, you will leave the classes in this pr, and once it merges to [soroban](https://github.com/stellar/java-stellar-sdk/tree/soroban), then the wip there in https://github.com/stellar/java-stellar-sdk/pull/492 will refer to them?